### PR TITLE
Add checks for pillar.get and grains.get

### LINKED ISSUE #57: Feature request: Capture wrong pillar.get
I'd like to see a function implemented where usage of wrong `pillar.get` is detected and reported.

For example:
**wrong**:
```code
pillar.get('pillar:item')
```

**good**:
```code
salt['pillar.get']('pillar:item')
```

```code
pillar['pillar']['item']
```

This limits itself to state files, above "wrong" examples would most likely still work in pillarstack.

### PR DESCRIPTION
**Pull request by jbouter**
- Trigger on faulty grains.get
- Trigger on faulty pillar.get
- UnitTests are included

Fixes #57 and #58

Signed-off-by: Jeffrey Bouter <jeffrey@kn0x.org>

### CODE REVIEW

File: saltlint/rules/JinjaPillarGrainsGetFormatRule.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import re
+
+
+class JinjaPillarGrainsGetFormatRule(SaltLintRule):
+    id = '211'
+    shortdesc = 'pillar.get or grains.get should be formatted differently'
+    description = "pillar.get and grains.get should always be formatted" \
+                  " like salt['pillar.get']('item') or grains['item1']"
+    severity = 'HIGH'
+    tags = ['formatting']
```
Comment (roaldnefs) [inline saltlint/rules/JinjaPillarGrainsGetFormatRule.py]: By now there are several Jinja checks, so we might want to introduce a new tag called `jinja` to group them:

```suggestion
    tags = ['formatting', 'jinja']
```

File: saltlint/rules/JinjaPillarGrainsGetFormatRule.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    version_added = 'develop'
+
+    bracket_regex = re.compile(r"{{ ?(pillar|grains).get\(.+}}")
```
Comment (roaldnefs) [inline saltlint/rules/JinjaPillarGrainsGetFormatRule.py]: The `{{` might also be followed by ` `, `-` and `+`.

File: saltlint/rules/JinjaPillarGrainsGetFormatRule.py (hunk 1 above)
Comment (jbouter) [reply saltlint/rules/JinjaPillarGrainsGetFormatRule.py]: The `jinja` tag has been added

File: saltlint/rules/JinjaPillarGrainsGetFormatRule.py (hunk 2 above)
Comment (jbouter) [reply saltlint/rules/JinjaPillarGrainsGetFormatRule.py]: I have updated the regex. Please see whether you like this solution and resolve if you do.

**Pull request merged**